app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+end
```

Comment:
- [ ]  As requested by the project requirements, the Like model should have a method that updates the likes counter for a post. Kindly add this.

Reply:
Thanks